mypy-requirements.txt

```diff
@@ -1,4 +1,4 @@
-typing_extensions>=3.7.4
+typing_extensions>=3.10
```

Comment:
Or maybe we need to exclude https://pypi.org/project/typing-extensions/4.0.0/ just yet? 🤔

Reply:
We don't need to, there are no incompatible changes (that mypy cares about).